azurerm/internal/services/sql/tests/resource_arm_sql_database_test.go

```diff
@@ -418,6 +418,36 @@ func TestAccAzureRMSqlDatabase_bacpac(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMSqlDatabase_updateWithBlobAuditingPolices(t *testing.T) {
```

Comment:
COuld we add a test with a single step config `testAccAzureRMSqlDatabase_withBlobAuditingPolices`